ui/src/index.ts

```diff
@@ -34,5 +36,6 @@ export * from './wizard/ResourceGroupListStep';
 export * from './wizard/StorageAccountCreateStep';
 export * from './wizard/StorageAccountListStep';
 export * from './wizard/StorageAccountNameStep';
-export { createAzExtOutputChannel } from './AzExtOutputChannel';
-export { registerUIExtensionVariables } from './extensionVariables';
+export * from './wizard/VerifyProvidersStep';
+
+// NOTE: The auto-fix action "source.organizeImports" does weird things with this file, but there doesn't seem to be a way to disable it on a per-file basis so we'll just let it happen
```

Comment:
I gave up in ARM a while ago.  Might consider disabling no-consecutive-blank-lines as well.  See https://github.com/microsoft/vscode-azurearmtools/blob/master/extension.bundle.ts